filter_plugins/cisco_nxos.py

```diff
@@ -0,0 +1,443 @@
+import re
+
+def getVLANList(config):
+    """
+    Parses vlan ranges in config files. For example, vlan 5-8,10,15-17 will return [5,6,7,8,10,15,16,17]
+
+    :param config: The config string block to parse the VLAN block for
+    :type config: str
```

Comment:
Type annotations are a great way to document the type of parameters:

```
def getVLANList(config: str):
    ...
```

(And allow things to be type-checked in your development environemnt.)